src/firecracker/src/main.rs

```diff
@@ -191,7 +191,8 @@ fn main() {
         // It's safe to unwrap here because the field's been provided with a default value.
         let level = arguments.value_as_string("level").unwrap();
         let logger_level = LoggerLevel::from_string(level).unwrap_or_else(|err| {
-            panic!("Invalid value for logger level: {}. Possible values: [Error, Warning, Info, Debug]", err);
+            error!("Invalid value for logger level: {}. Possible values: [Error, Warning, Info, Debug]", err);
+            process::exit(i32::from(vmm::FC_EXIT_CODE_GENERIC_ERROR));
```

Comment:
Would `FC_EXIT_CODE_ARG_PARSING` be more explicit?

Reply:
Hmm, I'm not so sure, when I added this code, I was thinking that it makes sense only for errors returned from the `ArgParser` processing. This `from_string` validation isn't necessary related to the actual argument parsing.

Reply:
Alright. We can use the generic code since the error is printed.